### PR TITLE
include code reviews section - Fixes #525

### LINKED ISSUE #525: Add Code Review to Contributing.md
- Add a "Code Reviews" section on CONTRIBUTING.md
- Put the following under the header:
> If you can, please look at open PRs and review them. Give feedback and help us merge these PRs much faster! If you don't know how, Github has some great [information on how to review a Pull Request](https://help.github.com/articles/about-pull-request-reviews/).
- Link the header in the Contributing.md Table of Contents
- Add a link to the README.md file: [Review Pull Requests]("CONTRIBUTING.md#Code-Reviews") under "Contributing Section"

### PR DESCRIPTION
**Pull request by jlax47**
# Fixes # 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Updates contributing.md to include a section on code review.

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=526) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=526) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-python?pullRequest=526) it.</sub>

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -229,3 +230,7 @@ Please run your code through:
     with a clear title and description against the `master` branch. All tests must be passing before we will review the PR.
 
 If you have any additional questions, please feel free to [email](mailto:dx@sendgrid.com) us or create an issue in this repo.
+
+<a name="code-reviews"></a>
+## Code Reviews
+If you can, please look at open PRs and review them. Give feedback and help us merge these PRs much faster! If you don't know how, Github has some great information on how to review a Pull Request.
```
Comment (delirious-lettuce) [inline CONTRIBUTING.md]: @jlax47 ,

I just noticed that the link from #525  is missing here.

* https://help.github.com/articles/about-pull-request-reviews/

Comment (jlax47) [reply CONTRIBUTING.md]: @delirious-lettuce,

Right you are. Sorry about that. I just updated it and pushed the changes into the same commit, it's showing up for me, can you confirm?

**Comment by thinkingserious** [comment]
Hello @jlax47,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Comment by mbernier** [comment]
@jlax47 

If you are interested, we would love the opportunity to talk to you about Hacktoberfest and our API libraries.

Our agenda would be: Explore what you liked and is there anything we can do to improve? 

You can grab a time [on my calendar](https://calendly.com/matt-bernier/hacktoberfest-follow-up) that works for you and we can have a chat on Google Hangout or Skype. If you prefer, you can email me using my GitHub username at my company’s domain.

Thank you so much,

Matt Bernier - @mbernier - SendGrid Developer Experience Product Manager
Elmer Thomas - @thinkingserious - SendGrid Developer Experience Engineer
@SendGrid

**Pull request merged**